src/content/en/2021/compression.md

```diff
@@ -2,7 +2,7 @@
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Compression
 description: TODO
-authors: []
+authors: [lvandeve]
 reviewers: []
 analysts: []
```

Comment:
@lvandeve can you suggest edits to the Authors, Reviewers, and Analysts properties so that they contain everyone who contributed to the chapter, roughly ordered by those with the most significant contributions first? This is what readers will see at the top of the chapter so we want to be sure to give appropriate credit to everyone who helped.

Reply:
Done, I've added the suggestion, there is one more person with significant review I'd like to add whose github username I need to check

Reply:
Added the extra reviewer as well now

Reply:
Ok thanks I'll add them in another PR

Reply:
https://github.com/HTTPArchive/almanac.httparchive.org/pull/2531